smoke-tests/build.gradle.kts

```diff
@@ -81,6 +79,8 @@ tasks {
 
     val shadowTask = project(":javaagent").tasks.named<ShadowJar>("shadowJar").get()
     inputs.files(layout.files(shadowTask))
```

Comment:
are these jar files? if so it might make more sense to use runtime classpath ?

Reply:
that's a good point, this is a Test task with an extra Jar file as input, my impression is that runtime classpath makes sense here.

Reply:
👍